assets/models/system/promptflow-chat-with-wikipedia/asset.yaml

```diff
@@ -0,0 +1,4 @@
+extra_config: model.yaml
+spec: spec.yaml
+type: model
+categories: ["Models"]
```

Comment:
This should be category of an asset to be properly grouped here https://github.com/Azure/azureml-assets/wiki/models-documentation. Like Bert,  GPT. Please also provide description

Reply:
Use "prompt flow" category to group prompt flow related modesl. Add description for these two models. We will refine the description before we decided to release these two model to prod registry. Here we only want to go through the e2e path in test registry.